src/assets/scripts/client/InputController.js

```diff
@@ -132,6 +133,7 @@ export default class InputController {
      * @method _init
      */
     _init() {
+        this.$body = $('body')[0];
```

Comment:
Can we use the root `$element` that is captured in `App` instead of looking for it manually here?  I can't see it in the diff, but be sure that `this.$body` is also included in the `constructor`.

Reply:
Added a `$body` property (initially null), then set to `this.$element[0]` in `_init()`.